packages/cfd/src/Containers/cfd-dbvi-onboarding.tsx

```diff
@@ -37,22 +36,21 @@ const SwitchToRealAccountMessage = ({ onClickOk }: { onClickOk: () => void }) =>
     </div>
 );
 
-const CFDDbviOnboarding = ({
-    account_status,
-    context,
-    disableApp,
-    enableApp,
-    fetchAccountSettings,
-    has_created_account_for_selected_jurisdiction,
-    has_submitted_cfd_personal_details,
-    is_cfd_verification_modal_visible,
-    is_virtual,
-    jurisdiction_selected_shortcode,
-    openPasswordModal,
-    toggleCFDVerificationModal,
-    updateAccountStatus,
-    updateMT5Status,
-}: TCFDDbviOnboardingProps) => {
+const CFDDbviOnboarding = observer(() => {
```

Comment:
Can't you wrap the component with observer where you are exporting it ?
That is cleaner IMO.

Reply:
The approach to use observer at export is not feasible and the props are not connected. This approach is fine.
According to the documentation, [https://mobx.js.org/react-integration.html#always-read-observables-inside-observer-components](http://url/). Its to reduce re-renders and optimising the performance
We will lose the component type, That's why we need to wrap the component definition itself with observer instead of the reference to the component definition 🙇🏻